test/functional/forall/reduce-basic/test-forall-basic-reduce.cpp.in

```diff
@@ -25,11 +25,17 @@
 #include "test-forall-basic-@REDUCETYPE@.hpp"
 
 //
-// Data types for reduction basic tests
+// Data types for core reduction basic tests
 //
-using ReductionDataTypeList = camp::list< int,
-                                          float,
-                                          double >;
+using CoreReductionDataTypeList = camp::list< int,
+                                              float,
+                                              double >;
+
+//
+// Data types for bitwise reduction basic tests
+//
+using BitwiseReductionDataTypeList = camp::list< int >;
```

Comment:
Could we add `unsigned int` to this default list?

Reply:
Sure thing. I'll have it up over the weekend.

Reply:
@willkill07 @MrBurmark unsigned int tests are added. Please re-review at your convenience.  Thanks.